internal/services/mssql/mssql_server_resource.go

```diff
@@ -146,12 +145,6 @@ func resourceMsSqlServer() *pluginsdk.Resource {
 			"minimum_tls_version": {
 				Type:     pluginsdk.TypeString,
 				Optional: true,
-				Default: func() interface{} {
-					if features.ThreePointOhBeta() {
-						return "1.2"
-					}
-					return nil
-				}(),
 				ValidateFunc: validation.StringInSlice([]string{
 					"1.0",
```

Comment:
since we introduced this default value in 3.0 as a security measure - can we instead add a value called `Disabled` and make users explicitly opt to make their production environments less secure? That means we'll need to transform this in the create/read etc to account for it